src/Raven.Server/Documents/PeriodicBackup/PeriodicBackupRunner.cs

```diff
@@ -185,7 +185,7 @@ public PeriodicBackupRunner(DocumentDatabase database, ServerStore serverStore)
                 using (_database.DocumentsStorage.ContextPool.AllocateOperationContext(out DocumentsOperationContext context))
                 using (var tx = context.OpenReadTransaction())
                 {
-                    var backupToLocalFolder = CanBackupUsing(configuration.LocalSettings);
+                    var backupToLocalFolder = PeriodicBackupConfiguration.CanBackupUsing(configuration.LocalSettings);
```

Comment:
If we refuse to backup something that was configured, we need to add an operations log to that.